<launch>

<include file="$(find camera)/launch/camera.launch"/>
<include file="$(find color_mask)/launch/yellow_mask.launch"/>
<include file="$(find yellow_lane_angle)/launch/yellow_lane_angle.launch"/>
<include file="$(find pid)/launch/pid_angle.launch"/>

<include file="$(find motor_speed)/launch/motor_speed.launch"/>

  <group ns="jetbot">
    <node pkg="jetbot_demos" name="lane_following_angle_only_node" type="lane_following_angle_only.py"/>
  </group>

</launch>
